server/traces/traces.go

```diff
@@ -16,6 +16,50 @@ type Trace struct {
 	Flat     map[trace.SpanID]*Span `json:"-"`
 }
 
+func NewTrace(traceID string, spans ...Span) Trace {
```

Comment:
I'll add a card to our backlog so we use this method instead of building the trace manually when converting from the otel format.